packages/stacks-svelte/package.json

```
@@ -0,0 +1,43 @@
+{
+    "name": "@stackoverflow/stacks-svelte",
+    "description": "Stacks Components built in Svelte",
+    "type": "module",
+    "version": "0.4.0",
+    "exports": {
+        ".": {
+            "types": "./lib/components/index.d.ts",
+            "svelte": "./lib/components/index.js"
+        },
+        "./actions": {
+            "types": "./lib/actions/index.d.ts",
+            "import": "./lib/actions/index.js"
+        }
+    },
+    "scripts": {
+        "build": "svelte-package -i src -o lib && rimraf test-utils.d.ts",
+        "build:dependencies": "echo 'Building stacks-classic because web-test-runner.config.mjs needs dist/css/stacks.css' && npm run build -w ../stacks-classic",
+        "format": "prettier --write .",
+        "lint": "svelte-check && eslint .",
+        "lint:fix": "eslint . --fix",
+        "test": "npm run build:dependencies && web-test-runner",
+        "test:skipbuild": "web-test-runner",
+        "test:watch": "npm run build:dependencies && web-test-runner --watch",
+        "storybook": "storybook dev -p 6006",
+        "storybook:build": "storybook build -o netlify/dist"
+    },
+    "peerDependencies": {
+        "@stackoverflow/stacks": "^2.8.4",
```

Should we tie the version to whatever is present in the monorepo?

```suggestion
        "@stackoverflow/stacks": "*",
```

---

That's what I thought too but Giamir pointed out stacks-svelte is meant to be a standalone package. So we need to tie it to the version of stacks-classic it actually supports. Otherwise if we were to release stacks-classic v3 at some point and someone downloaded stacks-svelte which targets v2 for stacks-classic, it would break since they would end up pulling v3.